Basics/README.rst

```diff
@@ -57,3 +57,18 @@ This set of tutorial notebooks will help you explore the basic properties of the
           </a>
 
      - `Phil Marshall <https://github.com/LSSTScienceCollaborations/StackClub/issues/new?body=@drphilmarshall>`_
+     
+     * - **Broker Tutorial**
```

Comment:
A better name is "Butler Tutorial". According to the [DM Glossary](https://confluence.lsstcorp.org/display/LKB/DM+Glossary#/), a broker is a "software system that ingests the Alert Stream and processes that information (e.g., filtering, cross-matching, categorizing). Outputs may include: processed streams of value-added Alerts, sorted lists."